src/Tenancy/Concerns/ResolvesTenants.php

```diff
@@ -18,7 +18,7 @@
 
 trait ResolvesTenants
 {
-    protected function resolver(): Resolver
```

Comment:
Why exactly is this necessary?

Reply:
Just made more sense if you have 2 resolvers that you use in 1 class to name them appropriately.